iphone/Classes/TiUIListView.m

```diff
@@ -1815,7 +1815,24 @@ - (void)scrollViewWillBeginDragging:(UIScrollView *)scrollView
         [self.proxy fireEvent:@"dragstart" withObject:nil withSource:self.proxy propagate:NO reportSuccess:NO errorCode:0 message:nil];
     }
 }
-
+-(void)scrollViewWillEndDragging:(UIScrollView *)scrollView withVelocity:(CGPoint)velocity targetContentOffset:(inout CGPoint *)targetContentOffset{
+    
+    if ([self.proxy _hasListeners:@"scrolling"]) {
+        NSMutableDictionary *eventArgs = [NSMutableDictionary dictionary];
+        NSString* swipeString = @"none";
+        if (velocity.y > 0){
+            swipeString = @"up";
+        }
+        if (velocity.y < 0){
+            swipeString = @"down";
+        }
+        if(swipeString !=(id)[NSNull null])
```

Comment:
If I initialise the swipeString to nil and do a nil check before firing the event, eventually I get a bad access exception. I left if initialised as 'none' and I do if (swipeString != @"none") instead. In that way it doesn't crash.